api/common/logs.go

```diff
@@ -0,0 +1,132 @@
+// Copyright 2016 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package common
+
+import (
+	"fmt"
+	"net/url"
+	"time"
+
+	"github.com/juju/errors"
+	"github.com/juju/loggo"
+
+	"github.com/juju/juju/api/base"
+	"github.com/juju/juju/apiserver/params"
+)
+
+// TODO(ericsnow) Fold DebugLogParams into params.LogStreamConfig.
```

Comment:
LogStreamConfig has AllModels, Sink (basically a label), MaxLookbackDuration and MaxLookbackRecords
DebugLogParams has 4 fields for including/excluding entities/modules, + Limit, Backlog, Replay, NoTail
 
So the only overlap is MaxLookbackRecords/Backlog - the other fields would need some amount of dev work to support them in logstream or debug-log.

I think it would make sense to do it at some point, but probably not right now. Want me to get rid of the comment?